packages/openapi-v3/test/unit/controller-spec/controller-decorators/param-decorators/param.test.ts

```diff
@@ -166,25 +143,26 @@ describe('Routing metadata for parameters', () => {
         .withOperationName('greet')
         .withParameter({
           name: 'names',
-          type: 'array',
-          items: {
-            type: 'string',
+          schema: {
+            type: 'array',
+            items: {
+              type: 'string',
+            },
           },
           in: 'query',
         })
         .build();
 
       expect(actualSpec.paths['/greet']['get']).to.eql(expectedSpec);
     });
-
```

Comment:
Please preserve this empty line between tests, same comments applies to other places below too.